pages/linux/flameshot.md

```diff
@@ -2,24 +2,32 @@
 
 > Screenshot utility with a GUI.
 > Supports basic image editing, such as text, shapes, colors, and imgur.
-> More information: <https://flameshot.js.org>.
+> More information: <https://flameshot.org>.
 
-- Launch flameshot in GUI mode:
+- Launch flameshot launcher window:
```

Comment:
```suggestion
- Launch flameshot with an interactive mode:
```

Although technical terms have their place, the description is best when the user can gather some information from it.